vmcnet/models/core.py

```diff
@@ -329,7 +329,98 @@ def __call__(self, x: Array) -> Array:  # type: ignore[override]
         return self.final_dense(x)
 
 
-class LogDomainResNet(Module):
+class SimpleSlater(flax.linen.Module):
```

Comment:
As discussed, this should be `Module` from `vmcnet.models.core` to handle typing issues